### PR TITLE
add meetup exercise, part of 48 in 24

### PR DESCRIPTION
**Pull request by kmarker1101**

### CODE REVIEW

**Comment by kmarker1101** [comment]
So I went with a 5 in difficulty, following along with Common Lisp and Racket. It did seem a little more difficult to me, but I could go either way?

File: exercises/practice/meetup/meetup.el [hunk 1]
```diff
@@ -0,0 +1,14 @@
+;;; meetup.el --- Meetup (exercism)  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;;; Code:
+
+
+(defun meetup (month year dow schedule)
```
Comment (BNAndras) [inline exercises/practice/meetup/meetup.el:8]: `dow` is a bit vague. Can we call it `dayofweek` or `weekday`? I'd also like to change the order so it's year month dow schedule. That'd probably be easier for students to parse and it lines up a bit better with the expected format for the return value

**Review by BNAndras — CHANGES_REQUESTED** [review]

**Comment by BNAndras** [comment]
Marking it a medium exercise is fine for now. We can always look at the completion rate and adjust the difficulty if folks are having trouble completing the exercise. Students do get more rep for solving a hard exercise, but they also might skip it if it's marked a hard exercise. My understanding is if we change the difficulty later to hard, the solutions completed while it was marked medium will still get the increased rep from solving it as a hard exercise. So there's not much of a downside to keeping it as a medium at least initially.

File: exercises/practice/meetup/meetup.el (hunk 1 above)
Comment (kmarker1101) [reply exercises/practice/meetup/meetup.el]: should be done. Thanks

**Review by BNAndras — APPROVED** [review]

**Pull request merged**